diagnostic.md

```diff
@@ -6,21 +6,22 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+  The main task you perform in the Router is defining and linking all the routes to
+  the app. An ember route is linked to a template
     ```
```

Comment:
This first part is right, the second thing I was looking for is that in a route, we specify how to load model data using the model hook in that template. 1/2